templating/embedding_controllers.rst

```diff
@@ -67,8 +67,9 @@ The ``recent_list`` template is perfectly straightforward:
     (e.g. ``/article/*slug*``). This is a bad practice. In the next section,
     you'll learn how to do this correctly.
 
-To include the controller, you'll need to refer to it using the standard
-string syntax for controllers (i.e. **bundle**:**controller**:**action**):
+To include the controller, you'll need to refer to it using the standard string syntax for controllers (i.e. **controllerPath**:**action**).
+
+Where controllerPath is a Full Qualified Class Name. (Notice double ``\\`` as a directory separator) and action is method in controller without the action postfix.
```

Comment:
I think you can remove this entirely, as the examples below illustrate this nicely